python/src/dynamixel_sdk/group_sync_write.py

```diff
@@ -40,41 +41,41 @@ def makeParam(self):
 
         self.param = []
 
-        for id in self.data_dict:
-            if not self.data_dict[id]:
+        for dxl_id in self.data_dict:
+            if not self.data_dict[dxl_id]:
                 return
 
-            self.param.append(id)
-            self.param.extend(self.data_dict[id])
+            self.param.append(dxl_id)
+            self.param.extend(self.data_dict[dxl_id])
 
-    def addParam(self, id, data):
-        if (id in self.data_dict): # id already exist
+    def addParam(self, dxl_id, data):
+        if dxl_id in self.data_dict:  # dxl_id already exist
             return False
 
-        if len(data) > self.data_length: # input data is longer than set
+        if len(data) > self.data_length:  # input data is longer than set
             return False
 
-        self.data_dict[id] = data
+        self.data_dict[dxl_id] = data
 
         self.is_param_changed = True
         return True
 
-    def removeParam(self, id):
-        if not id in self.data_dict: # NOT exist
```

Comment:
It appears to be a wrong or discouraged syntax. Maybe it could be interpreted as 
`if (not id) in self.data_dict` 
so checking if True or False is in `self.data_dict`

Reply:
@roncapat 

You're right